### ISSUE TITLE
Research AI Models for Career Coaching

### ISSUE THREAD

**Issue by dryanakiev** [codingburgas/2425-11-v-pp-group-assignment-group-one-iliyan-iliev#3]
Investigate AI/ML models for job matching and career coaching (e.g., OpenAI, scikit-learn, TensorFlow).

**Comment by GNIvanov21** [comment]
This model is developed to enhance recruitment efficiency while maintaining high accuracy in candidate evaluation - [LlamaFactoryAI](https://huggingface.co/LlamaFactoryAI/cv-job-description-matching)

**Comment by DAYusein211** [comment]
makhdoomnaeem/career_counselor

Career Counselor App
I’m here to guide you toward the perfect career based on your skills, interests, and experience.

**Comment by BADimov21** [comment]
https://ollama.com/todorov/bggpt
https://ollama.com/library/deepseek-r1
https://huggingface.co/LlamaFactoryAI/cv-job-description-matching

**Comment by AZKazakov21** [comment]
Use two different models:
- One that we will train using datasets in Azure ML for choosing career path
- One language model that we will use as coach and assistant

**Comment by VDVichev21** [comment]
- Training a datasets in Azure ML for choosing the career path
- Coach  and assistant
